airflow/www/static/js/dag/details/graph/index.tsx

```diff
@@ -150,7 +150,7 @@ const Graph = ({ openGroupIds, onToggleGroups, hoveredTaskState }: Props) => {
     setArrange(data?.arrange || "LR");
   }, [data?.arrange]);
 
-  const { data: datasetsCollection } = useDatasets({
+  const { data: datasetsCollection } = useAssets({
```

Comment:
Same here but maybe you want to do it in a separate PR?

```suggestion
  const { data: assetsCollection } = useAssets({
```

Reply:
Yes, this will be included in the "UI file change "in https://github.com/apache/airflow/issues/42307